jobs/config.json

```diff
@@ -6139,7 +6139,7 @@
       "--image-family=cos-beta",
       "--image-project=cos-cloud",
       "--provider=gke",
-      "--test_args=--ginkgo.focus=\\[Feature:StackdriverMonitoring\\]|\\[Feature:StackdriverCustomMetrics\\]|\\[Feature:StackdriverMetadataAgent\\] --minStartupPods=8",
+      "--test_args=--ginkgo.focus=\\[Feature:StackdriverMonitoring\\]|\\[Feature:StackdriverCustomMetrics\\] --minStartupPods=8",
```

Comment:
🤦‍♂️ someone changed generated config again, want to revert this part, or update generated configs?

we'll need a better pattern..

Reply:
I'll create a commit to fix it, is this the only one? :/

Reply:
yeah, everything else looks fine